lib/graphql/relay/relation_connection.rb

```diff
@@ -23,9 +23,21 @@ def cursor_from_node(item)
         end
       end
 
+      def record_exists?(record)
```

Comment:
why is this parameter called `record`? Should it be called `relation`, since we expect it to be a Relation? (or Dataset).